proto/ibc/lightclients/solomachine/v1/solomachine.proto

```diff
@@ -1,189 +0,0 @@
-syntax = "proto3";
-
-package ibc.lightclients.solomachine.v1;
```

Comment:
This file is now unused. I'm unsure if we need to save it for archival purposes (on cosmos/ibc maybe?). But the v1 was never used in production as it was non-functional at stargate release, so I'm not sure what use it would be to save it somewhere

Reply:
we will almost be able to come back and find it if we need it outside of vcs for some reaosn.